catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/src/test/java/org/codice/ddf/spatial/ogc/wfs/v110/catalog/source/WfsSourceTest.java

```diff
@@ -1608,11 +1608,7 @@ public void testSortingNoSortOrder() throws Exception {
 
   @Test
   public void testSortingNoSortProperty() throws Exception {
-    // if sort property is missing, throw UnsupportedQueryException
-    expectedEx.expect(UnsupportedQueryException.class);
-    expectedEx.expectMessage(
-        "Source WFS_ID does not support specified sort property null with sort order SortOrder[ASCENDING]");
-
+    // query is still valid even if sort property is missing
```

Comment:
note, this is a behavior change.  Initially I thought a missing sort property should cause an exception to be thrown, but there are valid scenarios where a query may not include a sort property (for example, and ID query) so don't fail this case.. .just ignore and continue.